cmd/protoc-gen-connect-go/main.go

```diff
@@ -410,13 +459,8 @@ func serverSignatureParams(g *protogen.GeneratedFile, method *protogen.Method, n
 		g.QualifiedGoIdent(method.Output.GoIdent) + "], error)"
 }
 
-func procedureName(method *protogen.Method) string {
-	return fmt.Sprintf(
-		"/%s.%s/%s",
-		method.Parent.Desc.ParentFile().Package(),
-		method.Parent.Desc.Name(),
-		method.Desc.Name(),
-	)
+func procedureConstName(m *protogen.Method) string {
+	return fmt.Sprintf("%s%sProcedure", m.Parent.Desc.Name(), m.Desc.Name())
```

Comment:
On face value, naming this "Procedure" feels weird to me - Protobuf doesn't have any concept of procedure. I now realize connect does, but I don't know, must have missed that too. I just worry people are going to not grok that this is just a path

Reply:
If this is a sticking point, let's discuss it in person.

Reply:
Discussed in person, Peter's ok with this as-is.